doc/source/notebooks/intro_to_gpflow2.pct.py

```diff
@@ -223,52 +223,68 @@ def simple_training_loop(model: gpflow.models.SVGP, epochs: int = 1, logging_epo
 # %% [markdown]
 # ## Monitoring
 #
-# We can monitor the training procedure using `tf.summary`. First we create a summary writer object through which we can write scalars and images.
+# We can monitor the training procedure using gpflow monitoring utilities.
+# For a more detailed tutorial see [GPflow monitoring notebook](./basics/monitoring.ipynb).
 
 # %%
-from intro_to_gpflow2_plotting import plotting_regression, summary_matplotlib_image
+from gpflow.monitor import (
+    ImageToTensorBoard,
+    ModelToTensorBoard,
+    ScalarToTensorBoard,
+    MonitorCollection,
+)
+
+samples_input = np.linspace(0, 10, 100).reshape(-1, 1)
+
+
+def plot_model(fig, ax):
+    mean, var = model.predict_f(samples_input)
+    num_samples = 10
+    samples = model.predict_f_samples(samples_input, num_samples)
+    ax.plot(samples_input, mean, "C0", lw=2)
+    ax.fill_between(
+        samples_input[:, 0],
+        mean[:, 0] - 1.96 * np.sqrt(var[:, 0]),
+        mean[:, 0] + 1.96 * np.sqrt(var[:, 0]),
+        color="C0",
+        alpha=0.2,
+    )
+    ax.plot(X, Y, "kx")
+    ax.plot(samples_input, samples[:, :, 0].numpy().T, "C0", linewidth=0.5)
+    ax.set_ylim(-2.0, +2.0)
+    ax.set_xlim(0, 10)
+
 
-samples_input = to_default_float(np.linspace(0, 10, 100).reshape(100, 1))
+output_logdir = enumerated_logdir()
+model_task = ModelToTensorBoard(output_logdir, model)
+image_task = ImageToTensorBoard(output_logdir, plot_model, "samples_image")
+elbo_task = ScalarToTensorBoard(output_logdir, lambda data=None: model.elbo(data).numpy(), "elbo")
```

Comment:
```suggestion
elbo_task = ScalarToTensorBoard(output_logdir, lambda data: model.elbo(data).numpy(), "elbo")
```
I don't think elbo() is going to work without data?